content/docs/catalogue/pack_variants.md

```diff
@@ -0,0 +1,79 @@
+
+| Base Unit | Pack Size | Description                                                                           | **Pack**     |
+|-----------|-----------|---------------------------------------------------------------------------------------|--------------|
+|           | 1         | No unit is configured so will display as "Unit" and pack size 1 so no number is shown | Unit         |
+|           | 10        | No unit is configured so will display as "Unit" and pack size is shown as multiplier  | 10 x Unit    |
+| Capsule   | 1         | Unit is configured "Capsule"  will be displayed and pack size 1 so no number is shown | Capsule      |
+| Capsule   | 10        | Unit is configured "Capsule"  will be displayed and pack size is shown as multiplier  | 10 x Capsule |
+
+
+If pack variants are defined then the matching pack variant is shown in the **Pack** column. 
+
+For example
+
+![pack variant stock view example](/docs/catalogue/images/pack_variant_stock_view_example.png)
+
+In above example Acetylisalicylic Acid has three batches in stock, two of them have defined pack variant that match stock pack size and one pack size does not match any pack variant (20). Ibuprofen and Amoxicillin don't have matching pack variants. Ibuprofen does not have defined unit and Amoxicillin unit is configured as 'Tab'.
+
+## Pack variants for entering stock batches
+
+If item has pack variants defined then stock entry view will allow you to select existing pack variant or enter custom pack size
+
+![pack variant entry example](/docs/catalogue/images/pack_variant_stock_entry_example.png)
+
+The [default pack variant](/docs/catalogue/pack-variants/#default-pack-variant) will always be pre selected for new bathes
+
+## Pack variants for item view
+
+When viewing items where stock lines are not visible, for example in item catalogue or when making internal order, pack variants replace the base unit. A drop down will be presented for items with pack variants and [default pack variant](/docs/catalogue/pack-variants/#default-pack-variant) will be pre selected. Quantities will adjust based on the selected pack variant.
+
+Here **Blister** was selected, see AMC, SOH and MOS 
+
+![pack variant entry example](/docs/catalogue/images/pack_variant_item_blister_selected.png)
+
+Here **Tab** was selected, see AMC, SOH and MOS 
+
+![pack variant entry example](/docs/catalogue/images/pack_variant_item_tab_selected.png)
+
+## Default Pack Variant
+
+Default pack variant would either be *last selected pack variant for this store, or if pack variant has not be selected yet, it would be the pack variant matching the most used pack size for this store 
+
+*last selected pack - this preference is saved in the client, for example the browser instance or Open mSupply client instance (it's not saved for user and store for the entire Open mSupply instance)
```

Comment:
```suggestion
Default pack variant would either be last selected pack variant for this store, or if pack variant has not be selected yet, it would be the pack variant matching the most used pack size for this store.

The last selected pack will be saved in the browser (e.g. Firefox, Chrome, etc) or Open mSupply client (desktop or mobile). It is not saved for the user and store.
```